app/store/charmstore-api.js

```diff
@@ -285,12 +285,8 @@ YUI.add('charmstore-api', function(Y) {
         with a response of >= 400.
     */
     getEntity: function(entityId, successCallback, failureCallback) {
-      var filters;
-      if (entityId.indexOf('bundle') > -1) {
-        filters = 'include=bundle-metadata&include=manifest&include=extra-info';
-      } else {
-        filters = '';
-      }
+      var filters = 'include=bundle-metadata&include=charm-metadata' +
+                    '&include=charm-config&include=manifest&include=extra-info';
```

Comment:
consolidating these together is the first step in making the GUI support bundle and charms in the same namespace. (future)